### PR TITLE
constants.ts keyword update

### PR DESCRIPTION
**Pull request by nachimehta**
Changed all instances of `var` to `const` per ECMAScript 6 usage guidelines.

### CODE REVIEW

**Review by Dessix — CHANGES_REQUESTED** [review]
Please recompile `dist` so I can merge your pull request.

**Comment by nachimehta** [comment]
Tidied up the commit message, and squashed commits. Dist is compiled. @Dessix

File: dist/screeps.d.ts [hunk 1]
```diff
@@ -379,6 +378,7 @@ declare class ConstructionSite extends RoomObject {
     remove(): number;
 }
 declare var Memory: Memory;
+declare var RawMemory: RawMemory;
```
Comment (Dessix) [inline dist/screeps.d.ts]: This line shouldn't be in this pull request.

File: dist/screeps.d.ts [hunk 2]
```diff
@@ -2,77 +2,76 @@
...
-declare var COLOR_WHITE: number;
-declare var COLORS_ALL: number[];
-declare var CREEP_SPAWN_TIME: number;
-declare var CREEP_LIFE_TIME: number;
-declare var CREEP_CLAIM_LIFE_TIME: number;
-declare var CREEP_CORPSE_RATE: number;
-declare var OBSTACLE_OBJECT_TYPES: string[];
-declare var ENERGY_REGEN_TIME: number;
-declare var ENERGY_DECAY: number;
-declare var CREEP_CORPSE_RATE: number;
```
Comment (Dessix) [inline dist/screeps.d.ts:70]: Cool, this can go in with the patch.

**Review by Dessix — APPROVED** [review]
That'll do it!

**Pull request merged**